blocks/test/fixtures/core-gallery.html

```diff
@@ -1,10 +1,10 @@
 <!-- wp:core/gallery -->
 <div class="blocks-gallery wp-block-gallery">
 	<figure class="blocks-gallery-image">
-		<img src="https://cldup.com/uuUqE_dXzy.jpg" alt="title" />
+		<img src="https://cldup.com/uuUqE_dXzy.jpg" alt="title" id="1" />
```

Comment:
I missed updating the fixture with `data-postId`, but looks like it should switch toan  attribute in comment instead.